shellsupport/bash/bash.go

```diff
@@ -19,6 +19,7 @@ func bashShellHandler(c credentials.AWSCredential, export bool) []string {
 	}
 
 	return []string{
+		fmt.Sprintf("%s AWS_PROFILE=%s;", flags, c.AWSProfile),
```

Comment:
As this is not a required parameter for the vast majority of applications the export / variable statement should not be set if the value is empty.